File: Python/1978.py
## 소수 찾기
import sys
input = sys.stdin.readline

n = int(input())
arr = list(map(int, input().split()))

ans = 0
for num in arr:
    flag = False
    if num == 1:
        continue
    for i in range(2, num):
        if num % i == 0:
            flag = True
            break

    if flag == False:
        ans += 1
        
print(ans)

File: JAVA/src/_1978.java
import java.io.*;

// 소수 찾기
public class _1978 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int n = Integer.parseInt(br.readLine());
        String[] input = br.readLine().split(" ");
        int ans = 0;
        for (String temp : input) {
            int data = Integer.parseInt(temp);
            for (int i = 2; i <= data; i++) {
                if (i == data) {
                    ans++;
                }

                if (data % i == 0) {
                    break;
                }
            }
        }
        bw.write(ans + "");

        bw.flush();
        bw.close();
    }
}
